og_access/og_access.module

```diff
@@ -0,0 +1,208 @@
+<?php
+
+/**
+ * @file
+ * Enable access control for private and public groups and group content.
+ *
+ * @TODO handle visibility change
+ * @TODO set group content visibility default to that of the group
+ * @TODO move grants/access to service
+ */
+
+use Drupal\Core\Config\Entity\ConfigEntityBundleBase;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\node\NodeInterface;
+use Drupal\og\Og;
+
+/**
+ * The access realm of group member.
+ */
+define('OG_ACCESS_REALM', 'og_access');
```

Comment:
defines could ultimately move to be `const` in an `OgAccess` class

Reply:
I've implemented it and sent a PR to @zerolab fork from mine.